frontend/demo/src/actions/channel/index.ts

```diff
@@ -1,20 +1,13 @@
 import {createAction} from 'typesafe-actions';
 import _, {Dispatch} from 'redux';
 
-import {doFetchFromBackend} from '../../api/airyConfig';
-
 import {
   Channel,
-  ChannelApiPayload,
-  ChannelsPayload,
-  channelsMapper,
-  channelMapper,
-  connectChannelApiMapper,
-  disconnectChannelApiMapper,
   ConnectChannelRequestPayload,
   ExploreChannelRequestPayload,
   DisconnectChannelRequestPayload,
-} from '../../model/Channel';
+  AiryHttpClient,
```

Comment:
I would drop the `Airy` prefix (It's a comment I leave in most pull requests now so I'm considering adding some naming convention guide)